src/pybind/mgr/orchestrator/module.py

```diff
@@ -2515,7 +2515,8 @@ def _upgrade_check_image_name(image: Optional[str], ceph_version: Optional[str])
     @_cli_write_command('orch upgrade check')
     def _upgrade_check(self,
                        image: Optional[str] = None,
-                       ceph_version: Optional[str] = None) -> HandleCommandResult:
+                       ceph_version: Optional[str] = None,
+                       no_mon_flags: bool = False) -> HandleCommandResult:
```

Comment:
Is this meant to be on the `upgrade start` command rather than `upgrade check`? It doesn't looks like upgrade check would have anything to do with mon flags

Reply:
You're right, addressed!